gdb/riscv-tdep.h

```diff
@@ -69,10 +69,13 @@ enum {
 
 #define RISCV_LAST_REGNUM (RISCV_NUM_REGS - 1)
 
+typedef enum { UNKNOWN, YES, NO } supported_t;
```

Comment:
I'm a bit worried about namespace conflicts here, as these are all common words and this is a header file.  Is GDB C++ code now?  If so, can you make this an "enum class"?